src/collapse/dtrace.rs

```diff
@@ -211,10 +231,10 @@ impl Folder {
     }
 
     fn finish<W: Write>(&self, mut writer: W) -> io::Result<()> {
-        let mut keys: Vec<_> = self.occurrences.keys().collect();
+        let mut keys: Vec<(&String, &usize)> = self.occurrences.iter().collect();
         keys.sort();
```

Comment:
So we're now relying on the fact that tuples are sorted in element order? I guess that's reasonable. May be worth adding a comment about it though.

Reply:
Also, we should probably add this optimization to the perf collapser as well.

Reply:
ja I split them out one at a time so it's easy to track and carry over at the end

Reply:
All migrated, I'm good with this for now :)